COPYRIGHT.txt

```diff
@@ -388,6 +408,41 @@ License: BSD-3-clause
 
 
 
+License: Unicode
```

Comment:
Licenses are listed alphabetically too, so this should go just above `License: Zlib`.